### DISCUSSION TITLE
Master public key for -f

### DISCUSSION THREAD

**Discussion by krafrijo** [Chia-Network/chia-blockchain#5001] [Support]
Hey,

i started creasting plots over the command line, and used my Master public key for -f (Farmer public key)

Now i cant´t use the plots i created for farming.

Is there any way to still use them, or are all my created plots broken?

Greetings

Jonny

**Comment by ghost** [comment]
If you farming on same machine, you don't have to use -f option. Was it Master or Farmer key? I am not really sure, but i think you have to use your Farmer public key.

For instace, my script look like this:
```
.\chia.exe plots create -k 32 -b 4300 -u 128 -r 4 -t X:\Chia -d Z:\Chia;
```
I am plotting on one machine and then farming on different one with same cert on both.

**Comment by sargonas** [comment]
Are you 100% certain you sent the right key to the plot command? If you are plotting on your farming system, you shouldn't have to explicitly call the key, it will inherit it automatically. If you called the key directly anyways and made a typo or used the wrong key, then you would have overridden they farmers key with an incompatible one. 

here's something you can do to verify this. If you look at the plotter log file for a recent plot made with the key you specified, at the top of the log file it will show the public key you used. Now start a new plot *without* specifying the public key and let the system handle it automatically. You don't have to let the plot finish, just let it start so the first few lines of the log are written. Then look at the first few lines for the public key it used and see if they match or are different. If different then it means you were not passing the right public key at the plot time, which is why it doesn't recognize them. If they are the same, then the problem lies elsewhere.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
